examples/simulator-lang-rps-2-rps/client.mjs

```diff
@@ -0,0 +1,38 @@
+import * as lang from '@reach-sh/simulator-lang';
+import * as assert from 'assert';
+
+const main = async () => {
+  console.log("Init Testing!")
+  const s = new lang.Scenario();
+  await s.init();
+  const pi = await s.pingServer();
+  const alice = s.participants[0];
+  const bob = s.participants[1];
+  const consensus = s.consensus;
+  // init Alice
+  await alice.init();
```

Comment:
I don't understand how this is possible... when do you assign her an account? Maybe I need to read the client lang implementation first

Reply:
if the user doesn't pass an account as an argument, the `Participant` will automatically get assigned an account